.appveyor.yml

```diff
@@ -57,3 +57,10 @@ test_script:
   - set PATH=%CD%\x64\%CONFIGURATION%;%PATH%
   - set FI_PROVIDER=sockets
   - scripts\runfabtests.cmd
+
+skip_commits:
+  files:
+    - man/*
+    - prov/opx/*
+    - prov/psm./*
```

Comment:
Does it match `psm`?

Reply:
In general regular expression, `.?` matches 0~1 character. Not sure if the rule is the same here.